components/overlay_trigger.tsx

```diff
@@ -11,10 +11,16 @@ import store from 'stores/redux_store.jsx';
 
 export {BaseOverlayTrigger};
 
-type Props = OverlayTriggerProps;
+type Props = OverlayTriggerProps & {
+    disabled?: boolean;
+};
 
 const OverlayTrigger = React.forwardRef((props: Props, ref?: React.Ref<BaseOverlayTrigger>) => {
-    const {overlay, ...otherProps} = props;
+    const {
+        overlay,
+        disabled,
```

Comment:
I'm slightly hesitant about adding new props to this component since it was supposed to be a transparent replacement for the one provided by react-bootstrap until we upgraded it (or removed it), but I suppose we could always keep our wrapper around it or just remove it in the future if necessary. This isn't a particularly complicated new prop to maintain anyway.

Reply:
Completely agree. We can eventually unwrap it and use `show=false`—I tried `OverlayTrigger` from 1.3 and it works as-is (though ref-less, it does forward contexts properly).